pkg/asset/installconfig/aws/validation.go

```diff
@@ -166,6 +166,122 @@ func validateAMI(ctx context.Context, config *types.InstallConfig) field.ErrorLi
 	return field.ErrorList{field.Required(field.NewPath("platform", "aws", "amiID"), "AMI must be provided")}
 }
 
+// filterNotAvaibleEIPErrors checks if a given EIP allocation ID exists.
+func filterNotAvaibleEIPErrors(allErrs field.ErrorList) []error {
+	var invalidAndNotFoundEIPList []error
+	for _, err := range allErrs {
+		// Handle specific AWS error for non-existent allocation ID
+		if strings.Contains(err.Error(), "InvalidAllocationID.NotFound") {
+			invalidAndNotFoundEIPList = append(invalidAndNotFoundEIPList, err)
+		}
+	}
+	if len(invalidAndNotFoundEIPList) > 0 {
+		return invalidAndNotFoundEIPList
+	}
+	return nil
+}
+
+func validateEIPAllocations(ctx context.Context, meta *Metadata, fldPath *field.Path, config *types.InstallConfig) field.ErrorList {
+	if config.Platform.AWS.EIPAllocations == nil || len(config.Platform.AWS.EIPAllocations.IngressNetworkLoadBalancer) == 0 {
```

Comment:
Below you do `config.Platform.AWS != nil` check but here you look at `config.Platform.AWS.EIPAllocations == nil`. It doesn't make sense to check for `config.Platform.AWS != nil` since that check is already performed in https://github.com/openshift/installer/pull/8204/files#diff-bab1b8072f369df53e27d65607145fb93caabd3a86fdefbcafb176e823726069L48-L50